src/main/java/com/nha/abdm/wrapper/common/models/VerifyOtp.java

```diff
@@ -0,0 +1,11 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.common.models;
+
+import lombok.Data;
+
+@Data
+public class VerifyOtp {
```

Comment:
when we have abbreviations in class then we use upper case: VerifyOTP

Reply:
Understood and modified it to VerifyOTP